third_party/libtock-drivers/src/usb_ctap_hid.rs

```diff
@@ -71,6 +71,24 @@ pub enum UsbInterface {
     VendorHid = 1,
```

Comment:
I set the wrong numbers here. They are supposed to be 1 and 2, respectively. Maybe we should even rename the enum to `UsbEndpoint`.

Reply:
I've updated the enum numbers.

So USB has 1:many for interfaces: endpoints. So we could keep the endpoint terminology for more of the code. However, at some point, the code makes the 1:1 relationship between interfaces:endpoints. So may as well do it now.
So the name is OK at the moment.